File: app/src/main/java/com/example/msimple/view/holders/TopicsHolder.java
package com.example.msimple.view.holders;

import android.support.annotation.NonNull;
import android.view.View;
import android.widget.TextView;

import com.example.msimple.R;
import com.example.msimple.data.vos.TopicVO;

public class TopicsHolder extends BaseHolder <TopicVO> {

    TextView tvTopicsTitle;
    TextView tvTopicsDescription;

    public TopicsHolder(@NonNull View itemView) {
        super(itemView);

        tvTopicsTitle = itemView.findViewById(R.id.tv_topic_title);
        tvTopicsDescription = itemView.findViewById(R.id.tv_topic_description);


    }


    @Override
    public void bind(TopicVO data) {

        tvTopicsTitle.setText(data.getTopicName());
        tvTopicsDescription.setText(data.getTopicDesc());

    }
}

File: app/src/main/java/com/example/msimple/view/holders/TopicsHolder.kt
package com.example.msimple.view.holders

import android.view.View
import android.widget.TextView

import com.example.msimple.R
import com.example.msimple.data.vos.TopicVO

class TopicsHolder(itemView: View) : BaseHolder<TopicVO>(itemView) {

    internal var tvTopicsTitle: TextView
    internal var tvTopicsDescription: TextView

    init {

        tvTopicsTitle = itemView.findViewById(R.id.tv_topic_title)
        tvTopicsDescription = itemView.findViewById(R.id.tv_topic_description)


    }


    override fun bind(data: TopicVO) {

        tvTopicsTitle.text = data.topicName
        tvTopicsDescription.text = data.topicDesc

    }
}
